dm/docs/RFCS/20220124_enhance_task_manageability.md

```diff
@@ -0,0 +1,127 @@
+
+- `command`: Specifies the operation that you want to perform on one or more resources, for example `create` , `get` , `update` , `delete` , `etc` .
+
+- `flags`: Specifies optional flags. For example, you can use the `--master-addr` flags to specify the address and port of the DM-Master server. note that we **not allow** any non-keyword arguments, all arguments must be specified as this: `--flag value`.
+
+### dmctl commands for Task
+
+| command | Full Syntax Example                                                                                                                           | Flags                                                                          | Description                                                              |
+| ------- | --------------------------------------------------------------------------------------------------------------------------------------------- | ------------------------------------------------------------------------------ | ------------------------------------------------------------------------ |
+| check   | `dmctl task check --config-file=dm.yaml --error-count=1 --warn-count=1`                                                                       | --config-file, --error-count, --warn-count                                     | check the task config yaml file.                                         |
+| create  | `dmctl task create --config-file="dm.yaml"`                                                                                                   | --config-file                                                                  | create a stopped task with config file.                                  |
+| update  | `dmctl task update --config-file="dm.yaml"`                                                                                                   | --config-file                                                                  | update a stopped task with config file.                                  |
+| delete  | `dmctl task delete --task-name="test" --yes`                                                                                                  | --task-name, --yes                                                             | delete a not running task and remove all meta data for this task.        |
+| get     | `dmctl task get --task-name="test" --output="new_task.yaml"`                                                                                  | --task-name, --output                                                          | show the task config in yaml format, also support output to file.        |
+| list    | `dmctl task list --status="running" --source="source1,source2"`                                                                               | --status, --source                                                             | list all tasks in current cluster, support filter by status and sources. |
```

Comment:
prefer `--stage="running"`

suppose we want to get `status` of tasks of stage `running`, it's wield to use `status --status=running`

although there's no such filter for task, but there's one for validation.